css/style.css

```diff
@@ -2,7 +2,142 @@
  * Add your custom styles below
  * 
  * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
+ * - Be organized, use comments and separate your styles into meaningful chunks
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+html {
+  font-family: "Mukta", sans-serif;
+}
+body {
+  margin: 0;
+}
+
+nav a {
+  text-decoration: none;
+  color: #845f50;
+  text-decoration: solid;
+  padding: 10px;
+  font-size: 18px;
+}
+
+#hortHR {
+  width: 60%;
+}
```

Comment:
There are a lot of minor formatting conventions being broken throughout this, which isn't the end of the world. If you haven't had any luck with tooling to help you there let me know and I can help you get set up (appreciate this work was done before I reviewed the last one and gave you that feedback, so please ignore this if you've already actioned this!)